colcon_core/event_handler/__init__.py

```diff
@@ -116,6 +116,9 @@ def format_duration(seconds, *, fixed_decimal_points=None):
     :returns: The string representation of the duration
     :rtype: str
     """
+    assert seconds >= 0.0, \
```

Comment:
Ah oops, my bad I shouldn't have said assert. This should raise `ValueError` so it doesn't get stripped if someone has `PYTHONOPTIMIZE` set.

Reply:
Another good point :wink: b2f2d492f6bc9d2a2951214667f364522447b6de

Reply:
And cover the new condition and line: 79432ce7bb3f10767b4c6221f0a550687723712e